ui/src/app/pages/runtimes-list.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import * as fp from 'lodash/fp';
+
+import { environment } from 'environments/environment';
+import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
+import { workspacesApi } from 'app/services/swagger-fetch-clients';
+import { getAccessToken } from 'app/utils/authentication';
+import { NavigationProps } from 'app/utils/navigation';
+import { withNavigation } from 'app/utils/with-navigation-hoc';
+import { ajaxContext, Environments } from 'terraui/out/Environments';
+
+const ajax = (signal) => {
+  const jsonLeoFetch = (path) =>
+    fetch(environment.leoApiUrl + path, {
+      signal,
+      headers: { authorization: 'bearer ' + getAccessToken() },
+    }).then((r) => r.json());
+  return {
+    Workspaces: {
+      list: () => workspacesApi().getWorkspaces(),
```

Comment:
Based on trying this with the Leo calls, I'm not sure this even works, but failing calls would produce noise, so this is an improvement over not implementing this function.